packages/api/src/external/commonwell/__tests__/cw.register-facility.test.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { faker } from "@faker-js/faker";
+import * as uuidv7_file from "@metriport/core/util/uuid-v7";
+import { OrgType } from "@metriport/core/domain/organization";
+import { CommonWell } from "@metriport/commonwell-sdk/client/commonwell";
+import { Organization } from "@metriport/commonwell-sdk/models/organization";
+import { CertificateResp } from "@metriport/commonwell-sdk/models/certificates";
+import { Facility, FacilityType } from "../../../domain/medical/facility";
+import { makeFacility } from "../../../domain/medical/__tests__/facility";
+import { createOrUpdateFacilityInCw } from "../command/create-or-update-cw-facility";
+import { buildCwOrgNameForFacility } from "../shared";
+import * as createOrUpdateCwOrg from "../command/create-or-update-cw-organization";
+import { Config } from "../../../shared/config";
+import * as api from "../api";
+
+let mockedFacility: Facility;
+let mockedOboFacility: Facility;
+let createOrUpdateCqOrganizationMock: jest.SpyInstance;
+
+beforeEach(() => {
```

Comment:
I know there's not a lot of tests here, but I'd try to use `beforeAll` whenever possible to avoid running unnecessary code during unit tests. Could split what would make sense to run before all tests on this file and what needs to run before each single tests.

These things compound over time! :)

Reply:
Done.